oauth2_http/java/com/google/auth/oauth2/IamUtils.java

```diff
@@ -125,4 +128,73 @@ private static String getSignature(String serviceAccountEmail, Credentials crede
     GenericData responseData = response.parseAs(GenericData.class);
     return OAuth2Utils.validateString(responseData, "signedBlob", PARSE_ERROR_SIGNATURE);
   }
-}
+
+  /**
+   * Returns an IdToken issued to the serviceAccount with a specified
+   * targetAudience.
+   *
+   * @param serviceAccountEmail the email address for the service account to get
+   *                            an Id Token for
+   * @param credentials         credentials required for making the IAM call
+   * @param transport           transport used for building the HTTP request
+   * @param targetAudience      the audience the issued ID token should include
+   * @param additionalFields    additional fields to send in the IAM call
+   * @return New IdToken issed to the serviceAccount.
+   * @throws IOException if the IdToken cannot be issued.
```

Comment:
Can we add a `@see` link to the developer docs for this API

Reply:
done